docs/docs/deploy/heroku.md

```diff
@@ -0,0 +1,170 @@
+* [sentry](https://sentry.io/welcome/) for error-tracking;
+* [logdna](https://logdna.com/) for log aggregation; and 
+* [hosted postgresql](https://www.heroku.com/postgres) as a data store.
+
+You can check the other add-ons heroku comes with out of the box [here](https://elements.heroku.com/addons).
+
+## Minimum Viable Deployment
+
+There are a couple of ways that we can deploy to Heroku - they maintain excellent docs on this [here](https://devcenter.heroku.com/categories/deployment). 
+The deployment mechanism we'll be focusing on is the github integration. We'll be building out a production-ready pipeline here. 
+We'll connect our app to a springboard based github repo and deploy the application with a database.
+
+One of the primary benefits to working in Springboard is simplicity of building/deploying. It's a single, stateless JVM 
+instance that's connected to a database. Most of the tricky first-time setup has already been done in the gradle build scripts. 
+As such, anything that can deploy a spring boot project can deploy a springboard project without difficulty. 
+
+As it happens, Heroku has terrific integration with Spring Boot so deploying a live, production-ready application is pretty simple.   
+
+I'm going to presume you've signed up to a Heroku account, if you haven't [then do so](https://signup.heroku.com/), login to the Heroku dashboard and get ready to go. 
+
+### Step 1: Create a Heroku App
+
+If you've just signed up to Heroku you'll be presented with an empty dashboard. On this page, click the `New` button and select `App`.
+
+[![heroku empty dashboard](/assets/images/heroku-deploy/heroku-empty-dashboard.png)](/assets/images/heroku-deploy/heroku-empty-dashboard.png)
+
+This will take you to the create application page:
+
+[![heroku create a new app page](/assets/images/heroku-deploy/heroku-create-app.png)](/assets/images/heroku-deploy/heroku-create-app.png)
+
+When there fill in the details for your application and hit `Create App`. This will take you on to your new app page:
+
+[![heroku newly created app](/assets/images/heroku-deploy/heroku-created-app.png)](/assets/images/heroku-deploy/heroku-created-app.png)
+
+Hooray! An app is born!
+
+### Step 2: Create a pipeline and deploy
+
+Now, our app doesn't actually do anything yet. Let's fix that.
+
+You'll be prompted to add this app to a pipeline at the top of the page. Click to create a new pipeline and fill in the details:
+
+[![heroku pipeline creation](/assets/images/heroku-deploy/heroku-create-pipeline.png)](/assets/images/heroku-deploy/heroku-create-pipeline.png)
+
+This will forward you on to the newly created pipeline:
+
+[![heroku newly created pipeline](/assets/images/heroku-deploy/heroku-pipeline-created.png)](/assets/images/heroku-deploy/heroku-pipeline-created.png)
+
+When here, click the prompt to `Connect to Github`. If you don't see this prompt then navigate to the `Settings` tab. Once on this page, search for your GitHub repo then connect.
```

Comment:
I'm pretty sure comma after "If you don't see this prompt"- but maybe not with then following directly?

If you don't see this prompt, then navigate to the ...